soen390_backend/models/Award.ts

```diff
@@ -0,0 +1,14 @@
+import * as yup from "yup";
+//Some types are to be changed later
+export const award_schema = yup
+    .object()
+    .shape({
+        awardID: yup.number().required(),
```

Comment:
I think the id should be of string type. Firebase generates ids with alphabetical characters in them.